src/components/landing_page/LandingPage.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Particles from 'react-particles-js';
+import TypeIt from 'typeit';
+import UserAuthNav from './UserAuthNav';
+import './LandingPage.scss';
+
+type Props = {
+};
+
+type State = {
+  particles_params: Object,
+};
+
+class LandingPage extends Component<Props, State> {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      particles_params: {
```

Comment:
Can you extract this in some sort of configuration file?
Just put it in the same directory level.